### ISSUE TITLE
3mf with multiple objects

### ISSUE THREAD

**Issue by AlbertMN** [ChristophSchranz/Tweaker-3#18]
Hey there - first of all, thanks so much for this awesome project @ChristophSchranz!

I've noticed that Tweaker accepts `3mf` files, but that it doesn't quite know how to handle multiple objects within one `3mf` file, so it just selects one and smart-rotates that one.

Would it be possible for it to rotate and return multiple objects?

Here's an example file;
[Multi-instance-control-box.zip](https://github.com/ChristophSchranz/Tweaker-3/files/7112145/Multi-instance-control-box.zip)

**Comment by ChristophSchranz** [comment]
Hi @AlbertMN,

Yes that's possible.

I expect the 3MF handler  in `ThreeMF.py` returns the objects in a list, but in `Filehandler.py` only the first 3MF object is passed. 
https://github.com/ChristophSchranz/Tweaker-3/blob/25331d6c775cfcdfcd24c4ec28f93120f23295a8/FileHandler.py#L60-L63
The routine what happens with each object can be set in the file `Tweaker.py`.

I hope this answer helps you!

**Comment by AlbertMN** [comment]
> Hi @AlbertMN,
> 
> Yes that's possible.
> 
> I expect the 3MF handler  in `ThreeMF.py` returns the objects in a list, but in `Filehandler.py` only the first 3MF object is passed. 
> https://github.com/ChristophSchranz/Tweaker-3/blob/25331d6c775cfcdfcd24c4ec28f93120f23295a8/FileHandler.py#L60-L63
> The routine what happens with each object can be set in the file `Tweaker.py`.
> 
> I hope this answer helps you!

Awesome, thank you! Want me to test it out and do a PR, or are you on it?

**Comment by ChristophSchranz** [comment]
Yes @AlbertMN that would be great! But don't change the behavior of the object-loop in `Tweaker.py` for now.
